### ISSUE TITLE
A11y_.Net Core_Winforms_MainForm_Common Controls1_List View : Color contrast ratio of the selected and any non selected list item is less than 3:1 i.e 1.139:1

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/winforms#9233]
### .NET version

Microsoft.Net SDK6.0.311 (x64)

### Did it work in .NET Framework?

No

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev3;#WCAG1.4.11;#DesktopApp;#Win11;#Benchmark;

### Actual:
The Color contrast ratio Between the Selected and Non-Selected List item is less than 3:1 i.e 1.139:1

### Expected:
The Color contrast ratio should be 3:1 minimum for the selected and Non Selected List Item.

### User Impact:
Low vision users will face difficulty in differentiating between the Selected and Non Selected List item.

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Tab till "Common Controls_1: Testing the controls under common controls Tab" and Hit enter.
4. Navigate to "List View" Section
5. Observe with (Accessibility Insights for web)Color contrast analyzer that the Ratio of the selected and Non-Selected List Item is less than 3:1.

Attachment : 
![Screenshot (3)](https://github.com/dotnet/winforms/assets/85863895/1c2e12a0-dfae-47b5-bf48-cdab3f3fa316)

**Comment by merriemcgaw** [comment]
@dmitrii-drobotov @LeafShi1 can your teams take a look and see if we can set the selection color in our code, or if this is solely done by windows in the Common Controls ListView control. I'm thinking it's most likely the ListView and we can't change it so we'd need an exception.

**Comment by SimonZhao888** [comment]
@merriemcgaw , we asked Dmitrii about this, and find that there is no method we can use for that and it's on the Windows control side.

**Comment by merriemcgaw** [comment]
Thanks @SimonZhao888 I think our existing exception around Common Controls should cover this case. I will review with the accessibility team.

**Comment by 12-shweta** [comment]
#A11yWCAG2.1;

**Comment by kupatkar99** [comment]
GithubTags:#ResolvedByTT; #External;

**Comment by merriemcgaw** [comment]
@kupatkar99 do the tags you just added indicate that the WinForms team is free to close this issue since there's nothing we can do from our end? We have the appropriate VPAT documentation and the exception for legacy Win32 Common Controls
